tfhe/src/typed_api/booleans/client_key.rs

```diff
@@ -0,0 +1,25 @@
+use crate::boolean::client_key::ClientKey;
+
+use serde::{Deserialize, Serialize};
+
+use super::{parameters::BooleanParameterSet, FheBoolParameters, types::r#static::StaticBoolParameters};
+
+ #[cfg_attr(doc, cfg(feature = "boolean"))]
```

Comment:
format issue I think (?)

How did lints not catch this 🤔

Reply:
There is something preventing our checks and cargo fmt to look into the typed_api module i think
as when I run make fmt, nothing seems to changes when I would expect some changes

Reply:
Ok I saw that there was a `#![cfg_attr(fmt, rustfmt::skip)]` in typed_api/mod.rs once the review is done I will remove it and format